### PR TITLE
Enable Windows 7 IIS tests

### PR DESCRIPTION
**Pull request by natemcmaster**
Resolves https://github.com/aspnet/AspNetCore-Internal/issues/1942

Waiting on https://github.com/dotnet/core-eng/issues/5506, which should be resolved soon.

### CODE REVIEW

**Review by jkotalik — APPROVED** [review]
As long as it works!

**Review by jkotalik — CHANGES_REQUESTED** [review]
This is still currently skipping all tests on Win7. It looks like IIS Express and IIS are still not installed on the machines.
```
C:\Program Files (x86)\IIS Express\config\schema\ doesn't exist
C:\Program Files\IIS Express\config\schema\ doesn't exist
C:\Program Files\IIS Express\config\schema\ doesn't exist
C:\windows\system32\inetsrv\config\schema\ doesn't exist
```

```
Updated C:\Program Files (x86)\IIS Express\config\schema\\aspnetcore_schema.xml
Updated C:\Program Files\IIS Express\config\schema\\aspnetcore_schema.xml
Updated C:\Program Files\IIS Express\config\schema\\aspnetcore_schema.xml
Updated C:\Windows\system32\inetsrv\config\schema\\aspnetcore_schema.xml
```

This means the config folders don't exist, which are always present when IIS/IISExpress is installed.

This seems like an issue with the win7 agents.

**Comment by natemcmaster** [comment]
> Waiting on dotnet/core-eng#5506, which should be resolved soon.

Was waiting on https://dev.azure.com/dnceng/internal/_git/dotnet-helix-machines/pullrequest/1228?_a=overview, but that was merged yesterday, so I'll re-spin the PR checks now.

**Comment by natemcmaster** [comment]
All tests were skipped still, so apparently queue updates haven't rolled out yet.

File: eng/targets/Helix.Common.props [hunk 1]
```diff
@@ -33,8 +33,7 @@
 
     <HelixAvailableTargetQueue Include="Windows.10.Amd64.EnterpriseRS3.ASPNET.Open" Platform="Windows" />
     <HelixAvailableTargetQueue Include="Windows.81.Amd64.Open" Platform="Windows" />
-    <!-- Disabled until https://github.com/dotnet/core-eng/issues/5506 is resolved -->
-    <!-- <HelixAvailableTargetQueue Condition="'$(IsWindowsOnlyTest)' == 'true'" Include="Windows.7.Amd64.Open" /> -->
+    <HelixAvailableTargetQueue Include="Windows.7.Amd64.Open" />
```
Comment (dougbu) [inline eng/targets/Helix.Common.props]: Let's hope you're right that dotnet/core-eng#5506 is done 😺

**Review by dougbu — APPROVED** [review]

**Comment by natemcmaster** [comment]
Helix update for Windows 7 is planned for today. Once that's done, I'll re-trigger PR checks to verify.

**Comment by natemcmaster** [comment]
All Windows.7.Amd64.Open machines should have IIS Express now. Re-running Helix tests to verify.

**Comment by jkotalik** [comment]
Now getting errors on sqllocaldb. I don't see any error messages either.

```
Executed on a001EDU

C:\dotnetbuild\work\f4958c6b-b3ed-4df3-97ea-823b7bd1fc8f\Work\a40e3bd3-6f53-441e-9eaa-720585794739\Exec>call RunPowershell.cmd mssql\InstallSqlServerLocalDB.ps1   || exit /b 1 
"PS: Running 'C:\dotnetbuild\work\f4958c6b-b3ed-4df3-97ea-823b7bd1fc8f\Work\a40e3bd3-6f53-441e-9eaa-720585794739\Exec\mssql\InstallSqlServerLocalDB.ps1' "
Installing SQL Server 2016 Express LocalDB

Downloading 'SqlLocalDB.msi' to 'C:\dotnetbuild\work\f4958c6b-b3ed-4df3-97ea-823b7bd1fc8f\Work\a40e3bd3-6f53-441e-9eaa-720585794739\Exec\mssql\obj\SqlLocalDB.msi'.

Running 'msiexec /package "C:\dotnetbuild\work\f4958c6b-b3ed-4df3-97ea-823b7bd1fc8f\Work\a40e3bd3-6f53-441e-9eaa-720585794739\Exec\mssql\obj\SqlLocalDB.msi" /NoRestart /Passive IACCEPTSQLLOCALDBLICENSETERMS=YES HIDEPROGRESSBAR=YES'.
```

**Comment by natemcmaster** [comment]
IISExpress tests for Win7 are running now

> Total tests: 271. Passed: 253. Failed: 0. Skipped: 18.

Didn't expect the SQL Server Local DB. They now pre-install this for us, but I guess our script doesn't no-op well if LocalDB is already installed.

**Review by jkotalik — APPROVED** [review]

**Pull request merged**